extensions/mssql/src/constants.ts

```diff
@@ -24,3 +24,8 @@ export const SqlMigrationService = 'sqlMigrationService';
 export const NotebookConvertService = 'notebookConvertService';
 export const AzureBlobService = 'azureBlobService';
 export const TdeMigrationService = 'tdeMigrationService';
+
+// COMMANDNAMES //////////////////////////////////////////////////////////
+export const cmdObjectExplorerGroupBySchemaFlagName = 'mssql.objectExplorer.groupBySchema';
+export const cmdObjectExplorerEnableGroupBySchemaCommand = 'mssql.enableGroupBySchema';
+export const cmdObjectExplorerDisableGroupBySchemaCommand = 'mssql.disableGroupBySchema';
```

Comment:
Should this one and the one above also have the form of 'mssql.objectExplorer.*' instead of 'mssql.*'

Reply:
I was following existing pattern in for command names

Reply:
I have separated the config name and named it appropriately.